gulp/tasks/gen-service-worker.js

```diff
@@ -89,9 +89,11 @@ gulp.task('gen-service-worker', () => {
         rootDir + '/fonts/roboto/Roboto-Bold.ttf',
         rootDir + '/images/card_media_player_bg.png',
       ],
-      stripPrefix: '..',
+      stripPrefix: 'hass_frontend',
       replacePrefix: 'static',
       verbose: true,
+      // Allow our users to refresh to get latest version.
+      clientsClaim: true,
```

Comment:
**Question:**
It sounds like this would mean there's a risk of serving an inconsistent set of resources, unless I'm misunderstanding:

https://www.npmjs.com/package/sw-precache#clientsclaim-boolean
https://developer.mozilla.org/en-US/docs/Web/API/Clients/claim

Reply:
Since we only use our service worker to serve cached files, it is ok. The benefit of using `claim` is that refreshing the page will help fix any caching issues because you will fetch the latest assets. Even better would be if we listen to `controllerchange` and refresh the page right away when a new service worker calls claim . It would mean that after upgrading Home Assistant, the first time you open the app, it will refresh after a couple of seconds (which could be I guess 20s on a slow connection).